realm-annotations-processor/src/main/java/io/realm/processor/RealmProcessor.java

```diff
@@ -127,6 +128,7 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
                 continue;
             }
             Utils.note("Processing class " + metadata.getSimpleClassName());
+            packages.add(metadata.getPackageName());
```

Comment:
The packages will have lots of duplicated entries. Use `Set` instead? And `getPackageName` doesn't work before calling `metadata.generate()` .